amd.test.ts

```diff
@@ -144,5 +152,38 @@ describe('Loader', function () {
     cleanUp();
   });
 
+  it('resolve relative deps', async () => {
+    const {System, fetcher} = setup();
```

Comment:
You can prob use `createFakeFetchableDef` here to save a line of code?

Reply:
I changed this test on another test for check bootstrap angular application